aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java

```diff
@@ -126,6 +128,12 @@
     @Parameter(property = "aem.analyser.classifier")
     private String classifier;
 
+    /**
+     * Alternatively analyzes the given list of content package files. 
```

Comment:
Please state more explicitly in the javadoc that this option will not be in addition to the attached artifacts but instead